addons/crm/views/crm_lead_views.xml

```diff
@@ -90,14 +90,20 @@
                                     </div>
                                 </div>
                                 <div class="col">
-                                    <span class="oe_edit_only o_pls_probability">
-                                        <label for="probability"/>
-                                        <button class="btn btn-link" name="action_set_automated_probability" type="object"
-                                                attrs="{'invisible': [('is_automated_probability', '=', True)]}">
-                                            <i class="fa fa-gear" role="img" title="Switch to automatic probability" aria-label="Switch to automatic probability"></i>
-                                        </button>
-                                        <small class="oe_grey" attrs="{'invisible': [('is_automated_probability', '=', True)]}"><field name="automated_probability"/> %% (Estimated by Odoo)</small>
-                                        </span>
+                                    <div class="oe_edit_only row o_pls_probability">
```

Comment:
While you are at it, try to remove o_pls_probability + its css and try to use small instead for field automated_probability and its tail text. That way we should be able to get rid of crm.css file.